docs/source/input.rst

```diff
@@ -46,4 +46,24 @@ from environment variables. To exclude internal stacks (like those responsible f
 Nivio proprietary format
 ------------------------
 
-Nivio provided an own format, which allows to set all model properties manually (see Model and Syntax section)
\ No newline at end of file
+Nivio provided an own format, which allows to set all model properties manually (see Model and Syntax section).
+
+
+External data
+-------------
+
+Nivio can load external data that can not be used directly to build landscapes, but is still valuable. For example, the
+number of GitHub issues might be interesting to see on a landscape item that is an open source component. To attach such
+data to landscape components, use links having special known identifiers like "github" or "sonar".
+
```

Comment:
It must be "cannot"

Reply:
?

Reply:
"Nivio can load external data that **can not** be used directly to build landscapes, but is still valuable." -> "Nivio can load external data that **cannot** be used directly to build landscapes, but is still valuable. "

Reply:
done